bugbot/rules/webcompat_score.py

```diff
@@ -0,0 +1,88 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from typing import Any, Optional
+
+from bugbot import gcp
+from bugbot.bzcleaner import BzCleaner
+
+
+class WebcompatScore(BzCleaner):
+    def __init__(self):
+        super().__init__()
+        self.autofix_changes = {}
+        self.scored_bugs = {}
+
+    def description(self) -> str:
+        return "Update WebCompat score fields"
+
+    def filter_no_nag_keyword(self) -> bool:
+        return False
+
+    def has_default_products(self) -> bool:
+        return False
+
+    def get_autofix_change(self) -> dict[str, Any]:
+        return self.autofix_changes
+
+    def handle_bug(
+        self, bug: dict[str, Any], data: dict[str, Any]
+    ) -> Optional[dict[str, Any]]:
+        bug_id = bug["id"]
```

Comment:
The ID should be a string consistent with other rules, and this is what the parent class expects:
```suggestion
        bug_id = str(bug["id"])
```

We have an issue to change that (i.e., https://github.com/mozilla/bugbot/issues/1432).

Reply:
Fixed, although note that making the change as written would break things (because the bug number is an integer in `self.scored_bugs`).

Reply:
nit: It would be great to have `bug_id = str(bug["id"])`, so it will less likely to be messied when doing the refactoring in https://github.com/mozilla/bugbot/issues/1432 (it will match the search query).

You could use `bug["id"]` directly when using the ID with `self.scored_bugs`.

Reply:
Done.